components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManagementServiceImpl.java

```diff
@@ -713,4 +720,69 @@ private void updateIsUniqueFromScope(Map<String, String> properties, String scop
         properties.put(ClaimConstants.IS_UNIQUE_CLAIM_PROPERTY, String.valueOf(isUnique));
     }
 
+    /**
+     * Validates the shared profile value resolving method change for system claims.
+     *
+     * @param updatedLocalClaim  Updated local claim.
+     * @param existingLocalClaim Existing local claim.
+     * @param tenantId           Tenant ID.
+     * @throws ClaimMetadataException If the shared profile value resolving method change is invalid for system claim
+     *                                or updating value is unaccepted.
+     */
+    private void validateSharedProfileValueResolvingMethodChange(LocalClaim updatedLocalClaim,
+                                                                 LocalClaim existingLocalClaim, int tenantId)
+            throws ClaimMetadataException {
+
+        String updatedClaimProperty =
+                updatedLocalClaim.getClaimProperty(ClaimConstants.SHARED_PROFILE_VALUE_RESOLVING_METHOD);
+        String existingClaimProperty =
+                existingLocalClaim.getClaimProperty(ClaimConstants.SHARED_PROFILE_VALUE_RESOLVING_METHOD);
+
+        if (!StringUtils.isBlank(updatedClaimProperty)) {
```

Comment:
```suggestion
        if (StringUtils.isNotBlank(updatedClaimProperty)) {
```

Reply:
For better readability

Reply:
fixed in fixed in [9148662](https://github.com/wso2/carbon-identity-framework/pull/6247/commits/9148662693d40927702ebe53b2250979a60d8c6f)